### DISCUSSION TITLE
Release notes missing/out of date on website

### DISCUSSION THREAD

**Discussion by michael-k** [encode/django-rest-framework#8013] [Potential Issue] [answered]
I've noticed that the release notes for 3.12.3 and 3.12.4 are missing on https://www.django-rest-framework.org/community/release-notes/.  While 3.12.4 might just be due to its young age, 3.12.3 (and its release notes at https://github.com/encode/django-rest-framework/blob/master/docs/community/release-notes.md) has been around for 2 months.

I've always thought https://www.django-rest-framework.org/community/release-notes/ is updated automatically in which case the automation might be broken.  But maybe it's done manually and no one had time to do it?

**Comment by peterfarrell** [comment, marked as answer]
Looks like there is issue regarding this #8047
